### ISSUE TITLE
Test Lite Builds Also

### ISSUE THREAD

**Issue by jaswrks** [websharks/travis-ci#1]
The current tests for PHPUnit and CasperJS are run against the final build of the repo. However, that doesn't consider that we may also want to run a set of tests against the final build of the lite variation as well. cc @raamdev

**Comment by raamdev** [comment]
@jaswsinc Yes, definitely. I had the same thought earlier today before seeing this GitHub issue. We really have two separate builds that occur with each `phing build`: Comet Cache Pro and Comet Cache Lite. Since we always do two releases, testing one without the other doesn't make sense.

**Comment by raamdev** [comment]
Also, I consider this `high priority`. 

I just discovered today while attempting to test a change to the Lite version, that the Lite version no longer builds properly (i.e., installing the Lite build results in a fatal error). I'm not even sure what change to the codebase over the past few weeks introduced this problem, but that's definitely the sort of thing we need to catch early on.

**Comment by jaswrks** [comment]
Copy that. What's happening now is that the Lite and Pro versions are being built, but whenever our PHPUnit and CasperJS tests run they are only being run against the pro codebase and not the lite codebase.
